### PR TITLE
Improve initialization of plot size

### LINKED ISSUE #3468: embed_responsive_with_height.py starts out wrong size
ping @birdsarah @almarklein 

After resizing the window, the plot "pops" to a much larger size then is responsive

```
examples/embed/embed_responsive_with_height.py:
 starts out small and "pops" to full size on browser window resize
```

### PR DESCRIPTION
**Pull request by almarklein**
issues: fixes #3468

The problem described in #3468 was related to a resize event being handled before the plot was fully initialized. This is why there was a `this.el.parentNode is null` error in the console, causing the initial sizing to fail. The solution is to test for this situation and schedule a new event in a future event-loop iteration.

### CODE REVIEW

File: bokehjs/src/coffee/common/plot.coffee [hunk 1]
```diff
@@ -455,6 +455,16 @@ class PlotView extends ContinuumView
     # user-configurable in the future, as it may not be the right number
     # if people set a large border on their plots, for example.
 
+    # We need the parent node, because the el node itself collapses to zero
+    # height. It might not be available though, if the initial resize
+    # happened before Bokeh got properly initialized. If that happens, we
+    # will try again, but only once.
+    if not @.el.parentNode and not @_re_resized
+      that = this
+      setTimeout( (-> that.resize_width_height(use_width, use_height, maintain_ar)), 10)
```
Comment (mattpap) [inline bokehjs/src/coffee/common/plot.coffee]: You should use fat arrow (=>) instead `that = this` JS idiom.

**Comment by mattpap** [comment]
>  before the plot was fully initialized

I would say simply before added to the DOM. Otherwise you can't take measurements.

File: bokehjs/src/coffee/common/plot.coffee [hunk 2]
```diff
@@ -455,6 +455,16 @@ class PlotView extends ContinuumView
     # user-configurable in the future, as it may not be the right number
     # if people set a large border on their plots, for example.
 
+    # We need the parent node, because the el node itself collapses to zero
+    # height. It might not be available though, if the initial resize
+    # happened before Bokeh got properly initialized. If that happens, we
+    # will try again, but only once.
```
Comment (mattpap) [inline bokehjs/src/coffee/common/plot.coffee]: Can it be unsuccessful after one retry? If so, what then?

**Comment by almarklein** [comment]
> Can it be unsuccessful after one retry? If so, what then?

@mattpap I think it should be successful in one try, as everything should be settled in the next event loop iteration. But indeed we cannot know how that works out in the future. I updated to try multiple times as needed with increasingly large intervals.

**Comment by mattpap** [comment]
LGTM

**Comment by mattpap** [comment]
Is this intended for 0.11?

**Comment by bryevdv** [comment]
> Is this intended for 0.11?

No

**Pull request merged**